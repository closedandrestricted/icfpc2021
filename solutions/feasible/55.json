{"vertices":[[23,14],[22,24],[40,42],[24,50],[11,41],[23,69],[36,33],[22,39],[22,39],[32,44],[25,58],[30,33],[30,33],[40,38],[38,41],[14,59],[10,36],[14,27],[27,58],[20,51]]}