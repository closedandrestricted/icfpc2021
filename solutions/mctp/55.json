{"vertices":[[29,66],[25,57],[48,27],[8,67],[7,67],[19,39],[55,34],[6,56],[6,56],[16,61],[28,32],[6,66],[6,66],[8,55],[17,64],[17,34],[22,6],[32,6],[3,91],[9,83]]}