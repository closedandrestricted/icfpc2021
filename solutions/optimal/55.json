{"vertices":[[25,73],[17,79],[14,0],[7,77],[10,98],[38,6],[58,33],[5,88],[5,88],[15,83],[0,96],[5,78],[5,78],[15,73],[5,67],[11,96],[4,69],[13,65],[25,63],[21,54]]}