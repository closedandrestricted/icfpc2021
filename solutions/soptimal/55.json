{"vertices":[[14,0],[15,10],[2,92],[21,13],[15,31],[6,60],[10,98],[28,4],[24,24],[31,15],[9,71],[38,6],[34,26],[41,17],[27,35],[10,60],[0,96],[9,92],[58,33],[48,34]]}