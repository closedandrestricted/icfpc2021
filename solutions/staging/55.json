{"vertices":[[16,43],[9,36],[8,89],[39,26],[29,32],[22,61],[10,79],[29,21],[29,21],[31,32],[33,64],[21,27],[21,27],[31,22],[28,36],[22,65],[38,38],[46,44],[36,7],[32,16]]}