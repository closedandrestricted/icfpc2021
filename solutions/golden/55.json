{"vertices":[[14,0],[16,10],[3,91],[21,13],[15,31],[10,60],[10,98],[28,4],[24,24],[31,15],[9,71],[38,6],[34,26],[41,17],[27,35],[11,60],[0,96],[9,92],[58,33],[48,35]]}